24c16.h

```diff
@@ -8,7 +8,7 @@ typedef struct {
 	unsigned char address;
 } t_24c16;
 
-typedef struct {
+typedef struct __attribute__((packed)){
```

Comment:
Not a good idea. This moves the `float` field to a misaligned offset, and may crash with SIGBUS on some CPUs.

Reply:
What do you suggest as alternative ?? Add a stuff variable instead ?

Reply:
Is it possible to define the float inside the struct as the first item? This will reduce the padding to 1 byte. Then we can add one byte to the serial number to fill that padding byte.